bin/dartdoc.dart

```diff
@@ -266,6 +266,14 @@ ArgParser _createArgsParser() {
           "Generates `index.json` with indentation and newlines. The file is larger, but it's also easier to diff.",
       negatable: false,
       defaultsTo: false);
+  parser.addOption('ambiguous-reexport-scorer-min-confidence',
```

Comment:
This seems like a super long name for a flag. Is this even a flag that user's should toggle? Should we just decide on a good value for this?

If you need a flag for dev purposes, you can have a hidden option with the `hide: true` param.